civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1102,20 +1102,20 @@ function civihr_employee_portal_views_date_fields_to_timestamp(&$view, &$query,
       // iterating single conditon structure
       foreach ($conditions as &$condition) {
 
-        $operatorEqualsFormula = isset($condition['operator']) && $condition['operator'] =='formula';
+        $operatorEqualsFormula = isset($condition['operator']) && $condition['operator'] == 'formula';
         if ( !$operatorEqualsFormula ) continue;
 
         // iterating over the fields that will be modified
         foreach ($dateFieldsToTimestamp as $dateFieldToTimestamp) {
 
-          $conditionHasField = strpos( $condition['field'], $dateFieldToTimestamp ) !== false;
+          $conditionHasField = strpos($condition['field'], $dateFieldToTimestamp) !== false;
```

Comment:
Please change all booleans (in non auto-generated files) to uppercase. You can run `civilint` to find them easily